src/test/java/bio/terra/service/snapshotbuilder/SnapshotBuilderServiceTest.java

```diff
@@ -590,4 +593,71 @@ void notifySnapshotReady() {
         .snapshotReady(
             eq(id), anyString(), eq(snapshot.getName()), eq("No snapshot specification found"));
   }
+
+  @Test
+  void testGetRequestGroupMembers() {
+    SnapshotAccessRequestMembersResponse expected = new SnapshotAccessRequestMembersResponse();
+    expected.members(new ArrayList<>());
```

Comment:
If you're not using a fluent-style setter in a fluent expression, I think it's better to use the non-fluent setter.
```suggestion
    SnapshotAccessRequestMembersResponse expected = new SnapshotAccessRequestMembersResponse();
    expected.setMembers(List.of());
```
or
```suggestion
    SnapshotAccessRequestMembersResponse expected = 
        new SnapshotAccessRequestMembersResponse().members(List.of());
```

for the `when` below, you can use this object to generate the return value, to avoid code duplication
```
    when(iamService.getGroupPolicyEmails(requestModel.samGroupName(), IamRole.MEMBER.toString()))
        .thenReturn(expected.getMembers());
```